### PR TITLE
COMMERCE-1845 fixed required fields on address modal

### PR DESCRIPTION
**Pull request by fabio-d-m**
I've also removed some legacy unused component

### CODE REVIEW

File: commerce-theme-minium/commerce-theme-minium/src/css/minium/components/commerce-input.scss [hunk 1]
```diff
@@ -52,4 +52,12 @@
...
\ No newline at end of file
+}
+
+.required-mark {
+	color: $color-mood-bad;
+
+	&::after {
+		content: '*';
+	}
+}
```
Comment (ethanbustad) [inline commerce-theme-minium/commerce-theme-minium/src/css/minium/components/commerce-input.scss]: trailing newline again :) please remove

**Review by ethanbustad — CHANGES_REQUESTED** [review]

File: commerce-theme-minium/commerce-theme-minium/src/css/minium/components/commerce-input.scss (hunk 1 above)
Comment (fabio-d-m) [reply commerce-theme-minium/commerce-theme-minium/src/css/minium/components/commerce-input.scss]: Hello @ethanbustad , I run the source formatter but it didn't change anything. Should I do it manually?

Comment (fabio-d-m) [reply commerce-theme-minium/commerce-theme-minium/src/css/minium/components/commerce-input.scss]: done it, anyway

Comment (ethanbustad) [reply commerce-theme-minium/commerce-theme-minium/src/css/minium/components/commerce-input.scss]: Thanks, @FabioDiegoMastrorilli ! You're the best 🙇

**Review by ethanbustad — APPROVED** [review]

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
Tests may pass now that https://github.com/liferay/com-liferay-commerce/pull/1436 is merged in.

**Pull request merged**